src/main/java/org/chirvascristina/classesandobjects/autoservicetask/Garage.java

```diff
@@ -0,0 +1,8 @@
+package org.chirvascristina.classesandobjects.autoservicetask;
+
+public class Garage {
+    public String address;
+     public float surfaceM2;
+     public int capacity;
+
```

Comment:
Ar fii bine constructorul din main "autoDocGarage" sa fie in clasa Garage. In main sa fie creat doar obiectul.